pom.xml

```diff
@@ -48,7 +66,7 @@
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
     <maven.deploy.skip>true</maven.deploy.skip>
-    <project.scm.organisation>Activiti</project.scm.organisation>
+    <project.scm.organization>Activiti</project.scm.organization>
```

Comment:
organisation is correct too, this is just the American spelling 🙂

Reply:
yeah but rest of the pom uses organization so better use one spelling